events.md

```diff
@@ -7,4 +7,4 @@ navigation_order: 1
 
 These are the planned events for Monthly Music Hackathon. Check out each one for more details and to RSVP.
 
-For past events, check in the [Event Archive](/events/archive.html).
```

Comment:
Permalink added.

I've looked using localhost. I've also searched through the _site folder, but am unsure of how to check what's been output from the build process using it. Please elaborate.